src/oc_erchef/apps/oc_chef_wm/itest/oc_chef_wm_cookbook_artifact_SUITE.erl

```diff
@@ -125,11 +125,8 @@ db_round_trip(Config) ->
     ?assert(erlang:is_integer(RecIDFromDB)),
 
     %% and modulo a few things, it should be equal to what we inserted
-    ChecksumsFromDB = RecFromDB#oc_chef_cookbook_artifact_version.checksums,
     ModifiedRecFromDB = RecFromDB#oc_chef_cookbook_artifact_version{id = undefined,
-                                                                    server_api_version = ?API_MIN_VER,
-                                                                    %% they get reversed when saving
-                                                                    checksums = lists:reverse(ChecksumsFromDB)},
```

Comment:
Did some functionality change that does not require this anymore?

Reply:
It was causing a verify pipeline error (see commit "correct verify pipeline error").  I assume 12.x made some change that caused this to fail.